stdlib/database/sql/src/main/java/org/ballerinalang/database/sql/Constants.java

```diff
@@ -180,18 +180,29 @@
         public static final String EQUAL_SYMBOL = "=";
     }
 
+    /**
+     * Constants used when appending JDBC driver properties for the connection URL.
+     */
+    public static final class DatabaseProductNames {
```

Comment:
Can't we simply use `Database` or `DatabaseName` here? Having `Product` in the class name does not seem right.